src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonOffenderEventListener.kt

```diff
@@ -27,6 +27,10 @@ class PrisonOffenderEventListener(
         val hmppsDomainEvent = mapper.readValue(message, HMPPSDomainEvent::class.java)
         prisonerIepLevelReviewService.processReceivedPrisoner(hmppsDomainEvent)
       }
+      "prison-offender-events.prisoner.merged" -> {
+        val prisonerMergeEvent = mapper.readValue(message, PrisonerMergeEvent::class.java)
+        prisonerIepLevelReviewService.processMergedPrisoner(prisonerMergeEvent)
```

Comment:
Could be worth adding tests to `PrisonerIepLevelReviewServiceTest` and `PrisonOffenderEventListenerIntTest` for this?

Reply:
Done